resources/assets/js/features/user/components/Profile/Reviews/Review.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {localize} from 'react-localize-redux';
+import DateTimeHelper from 'app/helpers/DateTimeHelper';
+import {defaultUserPhoto} from 'app/services/PhotoService';
+import LangService from 'app/services/LangService';
+import * as lang from '../../../lang/Profile/Reviews/Review.locale.json';
+
+import "../../../styles/review.scss";
+
+class Review extends React.Component {
+    componentWillMount() {
+        LangService.addTranslation(lang);
+    }
+
+    getStars(mark) {
+        let stars = [];
+        for (let i = 0; i < mark; i++) {
+            stars.push(<i className="fa fa-star" aria-hidden="true" key={i}></i>);
+        }
+        return stars;
+    }
+
+    render() {
+        const {mark, children, user, date, translate} = this.props;
+        const commentDate = DateTimeHelper.getTimeFromCommentDate(date);
+
+        return (
+            <div className="row comment">
+                <div className="col-sm-2">
+                    <img className="img-responsive user-photo" src={user.photo ? user.photo : defaultUserPhoto}/>
+                </div>
+                <div className="col-sm-10">
+                    <div className="card card-comment">
+                        <div className="card-header comment-header">
+                            <strong>{user.full_name}</strong>&nbsp;
+                            <span className="text-muted">
+                                {(Number.isInteger(commentDate) && commentDate <= 7)
```

Comment:
avoid using such statements in templates, define variable with appropriate value or method that will return needed value